### PR TITLE
SISRP-25121 Adjust caching of MyActivities, MyTasks, MyClasses, MyChecklist, MyRegistrations

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.berkeley.edu/browse/SISRP-25121

Some lightweight adjustments before tackling My Academics. As usual, reviewing one commit at a time will probably be easiest.

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -241,7 +241,7 @@ cache:
     LiveUpdatesWarmer: <%= 4.minutes %>
     BackgroundJobsCheck: <%= 29.days %>
     MyAcademics::Merged: NEXT_08_00
-    MyActivities::Merged: <%= 8.hours %>
+    MyActivities::Merged: <%= 10.minutes %>
```
Comment (davidlimcheng) [inline config/settings.yml]: @raydavis - Dropping this from an 8-hour expiration to a 10-minute expiration seems like a very drastic change to me, but actually the addition of `FreshenOnWarm` is currently causing the merged APIs to be called every 2 minutes (or 4, after #5887 was merged) - do I have that correct?

Comment (raydavis) [inline config/settings.yml]: You have it absolutely correct, @davidlimcheng .

Comment (davidlimcheng) [inline config/settings.yml]: Beautiful!

**Comment by davidlimcheng** [comment]
Looks great, @raydavis.

**Pull request merged**